### DISCUSSION TITLE
How to change Processing Commands Order

### DISCUSSION THREAD

**Discussion by aramdata** [SixLabors/ImageSharp.Web#293] [Q&A] [answered]
Hi ,
i write my custom command processors for crop the image (special rectangle from original image) and register it , but need to call before " Resize Web Processor " .
for example original image dimension is 4000*3000 and i want to crop it (top :10 , left :10 , crop width :1000 , crop height : 800 ) and then resize it to (width : 250 , height : 200). if first Resize Web Processor call before than my custom crop processor throw exception (Crop rectangle should be smaller than the source bounds).
 how to set order of processor ?

**Comment by JimBobSquarePants** [comment, marked as answer]
Pass the commands in the order you want them to run. 

```
?crop=[YOUR_COMMAND]&width=250&height=200
```

**Reply by aramdata** [reply]
thanks sir , its work
